config/crd/bases/csi-baremetal.dell.com_deployments.yaml

```diff
@@ -13,13 +13,302 @@ spec:
     kind: Deployment
     listKind: DeploymentList
     plural: deployments
+    shortNames:
+    - csi
+    - csis
     singular: deployment
-  scope: Cluster
+  scope: Namespaced
```

Comment:
How these changes are related to platform support? Are they autogenerated?

Reply:
mistakenly committed